container-chains/pallets/authorities-noting/src/lib.rs

```diff
@@ -124,6 +122,31 @@ pub mod pallet {
         }
     }
 
+    #[pallet::storage]
```

Comment:
It would be good to have a root-based extrinsic that can change this? Just in case for whatever reason, this gets updated